pkg/client/smd.go

```diff
@@ -16,6 +16,10 @@ type SmdClient struct {
 	Xname string
 }
 
+func (c SmdClient) Init() {
+	c.Client = &http.Client{}
```

Comment:
Getting these errors when linting:
```
Executing target: lint
/root/go/bin/golangci-lint run --fix
pkg/crawler/main.go:48:2: structtag: struct field tag `json:"system_type,omitempty` not compatible with reflect.StructTag.Get: bad syntax for struct tag value (govet)
        SystemType           string              `json:"system_type,omitempty`           // System type of the Node
        ^
pkg/client/smd.go:20:2: SA4005: ineffective assignment to field SmdClient.Client (staticcheck)
        c.Client = &http.Client{}
        ^
make: *** [Makefile:74: lint] Error 1
```

Reply:
Not sure how I missed the first one, but I'm unsure what to do about the second one yet.

Reply:
See:
- https://staticcheck.dev/docs/checks/#SA4005
- https://stackoverflow.com/questions/73494229/ineffective-assignment-to-field-when-trying-to-update-a-struct-in-go

Looks like because `c SmdCLient` is not a pointer, changes to its members in the member functions will be lost ("ineffective").